apps/server/src/shared/infra/collaborative-storage/strategy/nextcloud.strategy.spec.ts

```diff
@@ -120,4 +125,99 @@ describe('NextCloud Adapter Strategy', () => {
 			).rejects.toThrow(NotFoundException);
 		});
 	});
+	const teamIdMock = 'teamIdMock';
+	describe('Delete Team from Nextcloud', () => {
+		beforeAll(() => {
+			jest
+				.spyOn(strategy.httpService, 'get')
```

Comment:
Cant the http service mocked with mechanism which was showed in the brown bag session? Because of hardcoded string an so on...

Reply:
seems to be difficult, for now it works